test/parallel/test-dns-lookupService-promises.js

```diff
@@ -6,7 +6,7 @@ const assert = require('assert');
 const dnsPromises = require('dns').promises;
 
 dnsPromises.lookupService('127.0.0.1', 22).then(common.mustCall((result) => {
-  assert.strictEqual(result.service, 'ssh');
+  assert(['ssh', '22'].some((x) => x === result.service));
```

Comment:
```suggestion
  assert(['ssh', '22'].includes(result.service));
```
Since this is just a direct comparison with the elements of the array, the [`Array.prototype.includes()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes) method can be used instead.

Reply:
@VoltrexMaster I've updated it according to your review.